### PR TITLE
05/19 Upcoming Releases Update

### PR DESCRIPTION
**Pull request by russjones**

### CODE REVIEW

**Review by r0mant — APPROVED** [review]

File: docs/pages/preview/upcoming-releases.mdx [hunk 1]
```diff
@@ -35,28 +35,37 @@ Kubernetes Operator.
...
+### Teleport 10.2
+
+Teleport 10.2 will bring host certificate support to Machine ID and
+Elasticsearch support to Database Access.
+
+| Version     | Date         | Description                |
+|-------------|--------------|----------------------------|
+| Release     | August 2022  | Good to go for production. |
+
 ## Teleport Cloud "Washington"
```
Comment (jimbishopp) [inline docs/pages/preview/upcoming-releases.mdx:55]:
```suggestion
## Teleport Cloud
```

File: docs/pages/preview/upcoming-releases.mdx [hunk 2]
```diff
@@ -35,28 +35,37 @@ Kubernetes Operator.
...
 
-The key areas of focus for Teleport cloud in early 2022 are:
+The key areas of focus for Teleport Cloud in summer 2022 are:
 - Ensuring the best routing and latency experience
 - Stronger outbound communications on cloud health and infrastructure changes
 - UX where the cloud offering differs from the on-prem offering
```
Comment (jimbishopp) [inline docs/pages/preview/upcoming-releases.mdx:60]:
```suggestion
- UX where the cloud offering differs from the on-prem offering
- Scheduled upgrades will limit downtime to non-peak hours
```

**Review by jimbishopp — APPROVED** [review]

File: docs/pages/preview/upcoming-releases.mdx [hunk 3]
```diff
@@ -35,50 +34,65 @@ Kubernetes Operator.
...
+### Releases
 
 Releases are ready for production use.
 
-- Releases `5.0.0` and `6.0.0` are major releases. We publish 4 major releases each year.
-  Read more about upgrades and compatibility [here](../setup/operations/upgrading.mdx).
-- Releases `5.1.0` are minor releases. They contain minor backwards-compatible improvements and backports.
-- Versions like `5.0.1` are quick patches. They contain backwards-compatible fixes.
+- Releases `5.0.0` and `6.0.0` are major releases. We publish 4 major releases
+  each year.  Read more about upgrades and compatibility
```
Comment (ptgott) [inline docs/pages/preview/upcoming-releases.mdx:93]:
```suggestion
  each year. Read more about upgrades and compatibility in [Upgrading a Teleport Cluster](../setup/operations/upgrading.mdx).
```

File: docs/pages/preview/upcoming-releases.mdx [hunk 4]
```diff
@@ -35,50 +34,65 @@ Kubernetes Operator.
...
+  [here](../setup/operations/upgrading.mdx).
```
Comment (ptgott) [inline docs/pages/preview/upcoming-releases.mdx:94]:
```suggestion
```
included in the previous suggestion

**Review by ptgott — APPROVED** [review]

**Pull request merged**